packages/manager/apps/container/src/container/nav-reshuffle/sidebar/utils.ts

```diff
@@ -215,45 +220,44 @@ if no curly brackets, that returns an array that contains only the path */
 export const splitPathIntoSegmentsWithoutRouteParams = (
   path: string,
 ): string[] => {
+  // eslint-disable-next-line no-useless-escape
```

Comment:
Does removing both `\` between square brackets break the feature ?
```suggestion
  const regex = /\/(?!{)[^/]+(\/(?!{)[^/]+)?/g;
```